src/Endatix.Api/Endpoints/Submissions/PartialUpdateByToken.PartialUpdateSubmissionByTokenRequest.cs

```diff
@@ -0,0 +1,37 @@
+namespace Endatix.Api.Endpoints.Submissions;
+
+/// <summary>
+/// Request model for partial update of a form submission by token.
+/// </summary>
+public class PartialUpdateSubmissionByTokenRequest
```

Comment:
Good suggestion, thanks. As we talked, I'll merge this PR so the code is available and add the tests in another. I'll consider your suggestion there.

Reply:
As I looked a bit more, it doesn't feel like being just a small optimization for the partial requests. If we introduce an interface for token requests, then it makes sense to introduce others for the requests by submission ID (they are actually even more endpoints).
Reusing PartialUpdateSubmissionRequest in PartialUpdateSubmissionByTokenRequest also doesn't quite fit because the former has the ID in it, which is not present in the latter.
I generally don't like there is so much duplication in the request fields and the validation logic but we better think what approach to apply generally for all the endpoints.